README.md

```diff
@@ -1,17 +1,31 @@
 # flytectl-setup-action
-Install and setup [flytectl](https://github.com/flyteorg/flytectl) for use in other actions
+
+This action sets up [flytectl](https://docs.flyte.org/projects/flytectl/en/stable/) for use in actions by:
 
 ## Usage
 
-Refer to the [action.yml](https://github.com/unionai/flytectl-setup-action/blob/master/action.yml)
-to see all of the action parameters.
+Refer to the [action.yml](https://github.com/unionai/flytectl-setup-action/blob/master/action.yml) to see all of the action parameters.
 
 ```yaml
 steps:
   - uses: actions/checkout@v2
   - uses: unionai/flytectl-setup-action@v0.0.1
     with:
-      version: '0.1.8' # The version of flytectl to download and use.
+      version: '0.2.21' # The version of flytectl to download and use. If not set then it will automatically install latest version
   - run: flytectl --help
 ```
 
+## Getting started Examle
+```bash
+steps:
+  - uses: actions/checkout@v2
+  - uses: unionai/flytectl-setup-action@v0.0.1
```

Comment:
Do we need to specify the version of the action . Can it not pick the latest available.
Otherwise every new action release, this documentation would have to  be updated.

Reply:
Yes by default it use latest version

Reply:
Ok can we then use the latest here in the docs instead of pinning to a version of the action